install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -195,7 +195,7 @@ spec:
 {{- if .Values.global.etcd.managed }}
       # In managed etcd mode, Cilium must be able to resolve the DNS name of
       # the etcd service
-      dnsPolicy: ClusterFirstWithHostNet
+      dnsPolicy: {{ .Values.global.operator.dnsPolicy | default "ClusterFirstWithHostNet" }}
```

Comment:
We only use `global` object if the underlying option is used in multiple charts, not sure if this one is such case.

Another point is related to existing comment for this option, will comment still stay true?

Reply:
Actually yes, this comment is still valid. Without this change Cilium is not able to resolve.

Reply:
thanks :+1: 

I think you have fairly good details in PR description, however, git commit messages are kind of limited in details. Git commit message is quite useful during review and in long run. Would you please add commit message ? :)

I think you are already aware about https://docs.cilium.io/en/latest/contributing/development/contributing_guide/, I just copy here for easy access.